distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -191,6 +191,11 @@ Search for the existence of items containing multiple words, such as `Text = app
 +
 Search using wildcards, such as `Text = foo*` : Returns items containing words like "food", "fool", etc..
 +
+Wildcards should only be used for single word searches, not for phrases.
++
+[WARNING]
+When searching with wildcards, do not include the punctuation at the beginning or the end of a word. For example, search for `Text = ca*` instead of `Text = -ca*` when searching for words like "cat", "-cat", etc..  and search for `Text = *og` instead of `Text = *og.` when searching for words like "dog", "dog.", etc..
```

Comment:
Yeah I don't think this really explains why they need to remove it. It also doesn't make sense to me. If we strip the punctuation, why does the search return no results? I feel like it should work off just searching the word?

Reply:
It only searches for the word itself, so including the punctuation throws it off. I will update

Reply:
Maybe we should discourage the use of wildcards with search phrases entirely. If I wanted dog*, wouldn't I just need to do an ILIKE query on "dog"? No wildcard even needed. Why would I do EQUALS "dog*" instead? Maybe we should only do wildcard when there's no search phrase? 🤔

Reply:
I like this suggestion.

Reply:
I can agree with this suggestion, but it is out of the scope here. I believe the confusion here is that `Text =` is valid here for both ILIKE and EQUALS queries. It is just meant to reference the search phrase that is entered, not specifically as an EQUALS query. These changes here are valid either way.